src/changes/changes.xml

```diff
@@ -617,52 +617,52 @@ The <action> type attribute can be add,update,fix,remove.
         <release version="48" date="2019-03-20" description="Release version 48">
             <action type="update" dev="chtompki">Run "apache-rat:check" as part of "mvn validate"</action>
             <action type="fix" dev="chtompki">Revert revert change in commons.scmPubUrl in Parent 47, use commons.packageId for osgi</action>
-            <action type="update">commons.build-plugin.version: 1.9 -> 1.10</action>
-            <action type="update">commons.release-plugin.version: 1.5 -> 1.6</action>
-            <action type="update">commons.spotbugs.version: 3.1.3 -> 3.1.6</action>
-            <action type="update">japicmp-maven-plugin: 0.12.0 -> 0.13.0</action>
+            <action type="update" dev="">commons.build-plugin.version: 1.9 -> 1.10</action>
```

Comment:
dev attribute is required ... it was also required in old schema but not verified ...

Reply:
Hello @slawekjaranowski 

Thank you for creating this PR and putting in the work.

How about making the `dev` attribute optional for 3.0.0-M2? Absent or "" is the same IMO. Otherwise, that's a lot of mindless editing of 20+ changes.xml files in Commons... :-(

Reply:
plugin will be works ... only checking xml will be break - you don't need change it
but it is good idea to make it optional